Sources/CryptoSwift/Rabbit.swift

```diff
@@ -64,12 +64,12 @@ final public class Rabbit {
         
         // Key divided into 8 subkeys
         var k = [UInt32](count: 8, repeatedValue: 0)
-        for var j = 0; j < 8; j++ {
+        for var j = 0; j < 8; j += 1 {
```

Comment:
It is. However, I left those and the `Bit`-related deprecations alone, since those may have different solutions depending on your design aesthetic.